### PR TITLE
MGDOBR-6767,677: Check delete button is disabled for wrong confirmation

### PR DESCRIPTION
**Pull request by jomarko**
This PR is a follow up task after discussion on other PR:
https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox-ui/pull/63#issuecomment-1141802492

### CODE REVIEW

File: src/app/components/DeleteModal/DeleteModal.test.tsx [hunk 1]
```diff
@@ -110,6 +110,26 @@ describe("Delete Modal component", () => {
...
+    });
+    await waitForI18n(comp);
+
+    expect(await comp.findByText(title)).toBeInTheDocument();
+    expect(comp.getByText("Delete")).toBeDisabled();
+
+    fireEvent.change(comp.getByLabelText(`Type ${testName} to confirm.`), {
+      target: { value: "WRONG_VALUE" },
+    });
+    expect(comp.getByText("Delete")).toBeDisabled();
```
Comment (sjamboro) [inline src/app/components/DeleteModal/DeleteModal.test.tsx:130]: What do you think about an addition of an assertion that the "Cancel" button is enabled?

Comment (jomarko) [reply src/app/components/DeleteModal/DeleteModal.test.tsx:131]: sure, good idea, pushed in a separate commit, thank you

**Review by sjamboro — APPROVED** [review]

**Comment by jomarko** [comment]
@vpellegrino just a friendly reminder, if you have a moment for a review

**Review by kelvah — APPROVED** [review]
LGTM. Thanks.

**Pull request merged**